system_test.go

```diff
@@ -9,20 +9,30 @@ import (
 func TestSystemCreate(t *testing.T) {
 	// Arrange
 	request := `{
-    "query": "",
+    "query": "mutation SystemCreate($input:SystemCreateInput!){systemCreate(input:$input){system{id,aliases,name,description,htmlUrl,owner{... on Group{alias,id},... on Team{alias,id}},parent{id,aliases,name,description,htmlUrl,owner{... on Group{alias,id},... on Team{alias,id}}}},errors{message,path}}}",
 	"variables":{
-
-    }
-}`
+		"input": {
+			"name": "PlatformSystem3",
+			"description": "creating this for testing purposes",
+			"ownerId": "Z2lkOi8vb3BzbGV2ZWwvVGVhbS83NzU",
+			"note": "hello world"
+		  }
+    }}`
 	response := `{"data": {
-
-}}`
+		{{ template "system_create_response" }}
```

Comment:
Would be simpler to do this and doesn't require new templates to be created
```suggestion
"systemCreate": {
  "system": {
     {{ template "system1_response" }}
  },
  "errors": []
}
```

Same goes for the Get and Update response data